x-pack/test_serverless/functional/test_suites/search/search_query_rules/search_query_rules_overview.ts

```diff
@@ -134,9 +134,8 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
         // give time for the ruleset to be created
         await pageObjects.common.sleep(400);
-        await pageObjects.svlCommonNavigation.sidenav.clickLink({
-          deepLinkId: 'searchQueryRules',
-        });
+        await browser.navigateTo('about:blank');
```

Comment:
the sleep plus this makes these reliant on specific amounts of latency, which will be prone to flaking. 

We should look into writing a utility function that does a retry until the ruleset is available, we could use the API for that.